chall03/vtran.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+func main() {
+	resp, err := http.Get("https://chall03.hive.fi/")
+	ifError(err)
```

Comment:
You shouldn't do that in Go, I know that if err != nil {} can be a bit repetitive, and makes the code longer. It doesn't respect the best usage of Go.